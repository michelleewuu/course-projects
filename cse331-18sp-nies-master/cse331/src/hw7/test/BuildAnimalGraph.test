# Test building a graph with data animal.tsv

# Load animal.tsv
LoadGraph animal animal.tsv

# Check if graph is correctly built
ListNodes animal
ListChildren animal Monkey
ListChildren animal Otter
ListChildren animal Penguin