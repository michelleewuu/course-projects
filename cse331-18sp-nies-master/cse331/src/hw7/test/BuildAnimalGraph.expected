# Test building a graph with data animal.tsv

# Load animal.tsv
loaded graph animal

# Check if graph is correctly built
animal contains: Monkey Otter Penguin
the children of Monkey in animal are: Otter(0.333) Penguin(1.000)
the children of Otter in animal are: Monkey(0.333) Penguin(0.333)
the children of Penguin in animal are: Monkey(1.000) Otter(0.333)